docs/how-to-guides/deployment-options.md

```diff
@@ -51,78 +52,58 @@ Note: You are also able to update the service configuration post initial deploym
 
+The table below outlines the parameters that can be configured through any of the 3 deployment options:
+
+| Parameter | Type | Required | Description | Default Value |
+| --- | --- | --- | --- | --- |
+| serviceName | string | Yes | Used to generate a name for each of the resources provisioned wherever a name is not specified. | N/A |
+| containerAppImageTag | string | Yes | The tag of the FHIR Converter image version to be pulled from MCR. Visit the [FHIR Converter MCR page](https://mcr.microsoft.com/en-us/product/healthcareapis/fhir-converter/tags) to see available image tags. | N/A |
+| location | string | Yes | The Azure region where the resources will be deployed. See [region availability](https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?products=key-vault,monitor,storage,container-apps) for the relevant resources to select a valid location. | N/A |
+| timestamp | string | No | A timestamp to append to each deployment name to make it unique. | current date-time in 'yyyyMMddHHmmss' format |
+| resourceGroupName | string | No | The name of the resource group where the resources will be deployed. | serviceName + "-rg" |
+| containerAppEnvName | string | No | The name of the Container Apps environment. | serviceName + "-app-env" |
+| containerAppName | string | No | The name of the Azure Container App to run the FHIR converter. | serviceName + "-app" |
+| minReplicas | int | No | The minimum number of replicas for the Azure Container App. | 0 |
+| maxReplicas | int | No | The maximum number of replicas for the Azure Container App. | 30 |
+| cpuLimit | string | No | The CPU limit for the Azure Container App. | "1.0" |
+| memoryLimit | string | No | The memory limit for the Azure Container App. | "2Gi" |
+| templateStoreIntegrationEnabled | boolean | No | Enable or disable template store integration for the FHIR Converter service. If disabled, only default templates can be used for conversion. If enabled, only custom templates can be used. | false |
+| templateStorageAccountName | string | No | The name of the storage account to store custom templates. | serviceName + "templatestorage" |
+| templateStorageAccountContainerName | string | No | The name of the blob container to store custom templates. | serviceName + "templatecontainer" |
+| applicationInsightsEnabled | boolean | No | Enable or disable Application Insights deployment for your service. Application metrics and request logs will be unavailable for any time period where Application Insights is disabled. | true |
+| keyVaultName | string | No | The name of the Key Vault to store secrets. | serviceName + "-kv" |
+| keyVaultUserAssignedIdentityName | string | No | The name of the user-assigned managed identity to be used to access secrets in the Key Vault. | serviceName + "-kv-identity" |
+| securityEnabled | boolean | No | Enable or disable security settings for the APIs. | false |
```

Comment:
This is great! 

One suggestion but please feel free to won't fix cause it is purely person to person - maybe we can put this under a separate sub section titled Parameters. Then the default settings and configurable settings might tie in with that more clearly. And the configurable settings can just be referring to the parameters if they want to change and the configuration-settings for more info.

Reply:
On the flip side, I can also see how having this detailed table before default settings might be too much for someone who doesn't really want the specifics at first. 
So I'll leave it up to you. :)

Reply:
Thanks, I think I prefer that flow as well. Will make the update.

Reply:
Would it make sense to add the parameters section below "Configurable settings" so that we get the best of both?

Reply:
Yeah that works too!